### DISCUSSION TITLE
how get data from azuracast database by direct command ?

### DISCUSSION THREAD

**Discussion by wezo3** [AzuraCast/AzuraCast#6388] [General]
Hello,
I need to make bash script to bring data from database directly on the root access to check it
and I see this command : 
./docker.sh db
to connect to database console but this will not work with automation bash script

I need command like this : 
mysql DB -e "SELECT id FROM table"

to get the data directly from one command

how can do this with azuracast ?

Regards

**Comment by BusterNeece** [comment]
This is likely not possible with our setup unless you spin up a new Docker image that can directly access the database. I'd recommend using our API instead to make administration calls: https://www.azuracast.com/api/
